### PR TITLE
Closed household new digit component integration

### PR DESCRIPTION
**Pull request by Shashwat12-egov**

### CODE REVIEW

File: packages/closed_household/lib/pages/closed_household_summary.dart [hunk 1]
```diff
@@ -53,20 +57,19 @@ class ClosedHouseholdSummaryPageState
                 showcaseButton: ShowcaseButton(),
               ),
               footer: DigitCard(
-                margin: const EdgeInsets.fromLTRB(0, kPadding, 0, 0),
+                // margin: EdgeInsets.fromLTRB(0, theme.spacerTheme.spacer2, 0, 0),
```
Comment (naveenr-egov) [inline packages/closed_household/lib/pages/closed_household_summary.dart:60]: remove commented code

**Review by naveenr-egov — CHANGES_REQUESTED** [review]
Resolve comments and fix publock files

**Review by naveen-egov — DISMISSED** [review]

**Comment by naveen-egov** [comment]
@coderabbitai  please approve the PR

**Review by naveenr-egov — APPROVED** [review]

**Comment by naveenr-egov** [comment]
Already merged with this https://github.com/egovernments/health-campaign-field-worker-app/pull/609

**Pull request closed without merging**